azero/contracts/most/lib.rs

```diff
@@ -745,6 +746,38 @@ pub mod most {
             Ok(())
         }
 
+        /// Transfer PSP22 tokens from the bridge contract to a given account.
+        ///
+        /// Can only be called by the contracts owner
+        #[ink(message)]
+        pub fn recover_psp22(
+            &mut self,
+            token: AccountId,
+            receiver: AccountId,
+            amount: u128,
+        ) -> Result<(), MostError> {
+            self.ensure_owner()?;
```

Comment:
Why by owner and not committee?

Reply:
It's the same thing. Owner is a multisig.